### ISSUE TITLE
eth_sendTransaction not working

### ISSUE THREAD

**Issue by ckeenan** [ethereum/cpp-ethereum#2233]
Using eth built from develop's latest commit, when calling eth_sendTransaction to send ether to another account, web3 returns `''`, there's no change in state. Here is what I see with verbosity at 9

```
 ⚡   08:08:30|miner3  Worker stopping 0.319977 s
 ⚡   08:08:30|eth  Start worker 0.219108 s
ℹ    08:08:30|eth  noteChanged: { pending , chain }
┉┅▶  08:08:30|<unknown>  Queued vaguely legit-looking transaction #543c0e2a…
  ℹ  08:08:30|<unknown>  New transaction 543c0e2a…{d8b21af4…/0$100+300000@10000000000000<-39ec3dbc… #0}
  ℹ  08:08:32|miner2  Completed #adea05b2… #69fb8502… 135728 1
  ℹ  08:08:32|miner2  Mined #d0bf88bd… (parent: #d17461b8… )
▣┅▶  08:08:32|miner2  Queuing block #d0bf88bd… for import...
▧ ◌  08:08:32|miner2  Expected trie root: 56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421
▧ ◌  08:08:32|miner2  Expected uncle hash: 1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347
▣┅▶  08:08:32|miner2  OK - ready for chain insertion.
▧ ◌  08:08:32|verifier0  Expected trie root: 56e81f171bcc55a6ff8345e692c0f86e5b48e01b996cadc001622fb5e363b421
▧ ◌  08:08:32|verifier0  Expected uncle hash: 1dcc4de8dec75d7aab85b567b6ccd41ad312451b948a7413f0a142fd40d49347
☍ ◌  08:08:32|eth  Attempting import of #d0bf88bd… ...
⚙ ◎  08:08:32|eth  Committing to disk: stateRoot #8e6903c8… = #8e6903c8… = f9017180a0ab8cdb808c8303bb61fb48e276217be9770fa83ecf3f90f2234d558885f5abf1a0b43efbb595efd0043aca412c296b6064456c48f9ec418b9a1a7283f75b44bdbb8080a0c03f7cb3c033ba740e22eea1634424e0b03644bd93d59a7db7f35db3010a541880a0191667c1c090900caa92cfa970e64882ede808a792acfae6df9b170cf9a5e644a00a9341b201bfb2dccdb1997c627f3b333310735d2a5cb6060a92e1e5efa961c7a0b408086f1a609b8c07852c1e9ef28e8a04bca335a15547b4f535aaa51fc85e2ba0de2d76d77327d8f463f2eb68ff027d5e22e1b72d1413cf51f5cb8a3fbdd3a21e80a083de30075f2e5873ead424025a1fda192f5ea3313226e99c4cdd4aafc7b50493a01b7779e149cadf24d4ffb77ca7e11314b8db7097e4d70b2a173493153ca2e5a0a0939a6b4dae9579776ce0249d7ab2dd888372ed5bd44db16321cf4a7ebe21c4d6a0030b568c42865deb798cd015993240a64467e4ec87236c9a468e8d23a0ace2bf80
⚙ ◎  08:08:32|eth  Committed: stateRoot #8e6903c8… = #8e6903c8… = f9017180a0ab8cdb808c8303bb61fb48e276217be9770fa83ecf3f90f2234d558885f5abf1a0b43efbb595efd0043aca412c296b6064456c48f9ec418b9a1a7283f75b44bdbb8080a0c03f7cb3c033ba740e22eea1634424e0b03644bd93d59a7db7f35db3010a541880a0191667c1c090900caa92cfa970e64882ede808a792acfae6df9b170cf9a5e644a00a9341b201bfb2dccdb1997c627f3b333310735d2a5cb6060a92e1e5efa961c7a0b408086f1a609b8c07852c1e9ef28e8a04bca335a15547b4f535aaa51fc85e2ba0de2d76d77327d8f463f2eb68ff027d5e22e1b72d1413cf51f5cb8a3fbdd3a21e80a083de30075f2e5873ead424025a1fda192f5ea3313226e99c4cdd4aafc7b50493a01b7779e149cadf24d4ffb77ca7e11314b8db7097e4d70b2a173493153ca2e5a0a0939a6b4dae9579776ce0249d7ab2dd888372ed5bd44db16321cf4a7ebe21c4d6a0030b568c42865deb798cd015993240a64467e4ec87236c9a468e8d23a0ace2bf80
⚙ ◎  08:08:32|eth  finalising enactment. current -> previous, hash is #d0bf88bd…
☍ ℹ  08:08:32|eth     Imported and best 10409704  (# 77 ). Has 0 siblings. Route: [#d17461b8…, #d0bf88bd…]
  ℹ  08:08:32|eth  100 blocks imported in 1 ms ( 75471.7 blocks/s)
```

**Comment by christianlundkvist** [comment]
I can confirm I'm seeing the same issue.

**Comment by debris** [comment]
fixed in #2404
